src/components/Table/Table.jsx

```diff
@@ -0,0 +1,351 @@
+		 */
+		isSorted: bool,
+		/**
+		 * The direction of the caret in the sorted column.
+		 */
+		sortDirection: string
+	},
+	getDefaultProps() {
+		return {
+			align: 'left',
+			hasCheckbox: false,
+			hasIcon: false,
+			hasButton: false,
+			isSorted: false,
+			sortDirection: 'up'
+		};
+	},
+	render() {
+		const {
+			children,
+			className,
+			align,
+			hasCheckbox,
+			hasIcon,
+			hasButton,
+			isSortable,
+			isSorted,
+			sortDirection
+		} = this.props;
+
+		return (
+			<th {...this.props} className={boundClassNames(
+				'&-cell', {
+				'&-align-left': align === 'left',
+				'&-align-center': align === 'center',
+				'&-align-right': align === 'right',
+				'&-hasCheckbox': hasCheckbox,
+				'&-hasIcon': hasIcon,
+				'&-hasButton': hasButton,
+				'&-isSortable': (isSortable === false ? isSortable : (isSorted || isSortable)),
+				'&-isSorted': isSorted,
+			}, className)}>
+				{isSorted ? (
+					<ul className={boundClassNames('&-isSorted-container')}>
+						<li className={boundClassNames('&-isSorted-title')}>{children}</li>
+						<li className={boundClassNames('&-isSorted-caret')}><CaretIcon className={boundClassNames('&-sort-icon')} direction={sortDirection} size={6}/></li>
+					</ul>
+				) : children}
+			</th>
+		);
+	}
+}));
+
+/**
+ * `Td` renders <td>.
+ */
+const Td = React.createClass(createLucidComponentDefinition({
+	propTypes: {
+		/**
+		 * Aligns the content of a cell. Can be `left`, `center`, or `right`.
+		 */
+		align: string,
+		/**
+		 * Should be `true` when the cell has a checkbox.
+		 */
+		hasCheckbox: bool,
+		/**
+		 * Should be `true` when the cell has an icon.
+		 */
+		hasIcon: bool,
+		/**
+		 * Should be `true` when the cell has a button.
+		 */
+		hasButton: bool,
+		/**
+		 * Should be `true` to render a right border.
+		 */
+		hasBorderRight: bool,
+		/**
+		 * Should be `true` to render a left border.
+		 */
+		hasBorderLeft: bool,
+		/**
+		 * Should be set to `true` on the second cell in a table where the first cell has a rowspan greater than 1.
+		 */
+		isAfterRowSpan: bool
+	},
+	getDefaultProps() {
+		return {
+			align: 'left',
+			hasCheckbox: false,
+			hasIcon: false,
+			hasButton: false,
+			hasBorderRight: false,
+			hasBorderLeft: false,
+		};
+	},
+	render() {
+		const {
+			className,
+			align,
+			hasCheckbox,
+			hasIcon,
+			hasButton,
+			rowSpan,
+			hasBorderRight,
+			hasBorderLeft,
+			isAfterRowSpan
+		} = this.props;
+
+		return (
+			<td {...this.props} className={boundClassNames(
+				'&-cell', {
+				'&-align-left': align === 'left',
+				'&-align-center': align === 'center',
+				'&-align-right': align === 'right',
+				'&-hasCheckbox': hasCheckbox,
+				'&-hasIcon': hasIcon,
+				'&-hasButton': hasButton,
+				'&-hasRowSpan': !_.isNil(rowSpan),
+				'&-hasBorderRight': hasBorderRight,
+				'&-hasBorderLeft': hasBorderLeft,
+				'&-isAfterRowSpan': isAfterRowSpan
+			}, className)} />
+		);
+	}
+}));
+
+/**
+ *
+ * {"categories": ["table"]}
```

Comment:
It does. It's made from `CaretIcon`. I think the name "made from" is confusing. Perhaps "composes" is a better way of describing it.